gallery/src/components/NxList/NxListPage.tsx

```diff
@@ -0,0 +1,284 @@
+              <NxTable.Cell>errorMessage</NxTable.Cell>
+              <NxTable.Cell>string</NxTable.Cell>
+              <NxTable.Cell>yes</NxTable.Cell>
+              <NxTable.Cell>Error message to display.</NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell> </NxTable.Cell>
+              <NxTable.Cell>onClick</NxTable.Cell>
+              <NxTable.Cell>function</NxTable.Cell>
+              <NxTable.Cell>yes</NxTable.Cell>
+              <NxTable.Cell>Function for retry.</NxTable.Cell>
+            </NxTable.Row>
+          </NxTable.Body>
+        </NxTable>
+      </section>
+
+      <section className="nx-tile-subsection">
+        <header className="nx-tile-subsection__title">
+          <h3 className="nx-h3">Component Breakdown</h3>
+        </header>
+        <NxP><NxCode>NxList</NxCode> comprises of several children components.
+          A reference of each component and where they must be placed is as follows:
+        </NxP>
+        <NxTable>
+          <NxTable.Head>
+            <NxTable.Row>
+              <NxTable.Cell>Component</NxTable.Cell>
+              <NxTable.Cell>Location</NxTable.Cell>
+              <NxTable.Cell>Details</NxTable.Cell>
+            </NxTable.Row>
+          </NxTable.Head>
+          <NxTable.Body>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>NxList</NxCode></NxTable.Cell>
+              <NxTable.Cell>Parent; Top-Level</NxTable.Cell>
+              <NxTable.Cell>This is the parent list class. By default, it has no bullets. In order to have a
+                bulleted list, the prop <NxCode>bulleted</NxCode> can be provided.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>NxList.Title</NxCode></NxTable.Cell>
+              <NxTable.Cell>Within <NxCode>NxList</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                This is the title of the list.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>NxList.Item</NxCode></NxTable.Cell>
+              <NxTable.Cell>Within <NxCode>NxList</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                This is the standard list element that is not clickable, does not respond to hover events, and
+                does not have selected and disabled states.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>NxList.Button</NxCode></NxTable.Cell>
+              <NxTable.Cell>Within <NxCode>NxList</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                This list element that is clickable, responds to hover events, and has selected and disabled states.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>NxList.Link</NxCode></NxTable.Cell>
```

Comment:
I'd be in favor of calling these `NxList.LinkItem` and `NxList.ButtonItem` or `NxList.ClickableItem` to make it clear that they encapsulate the `<li>` and that you shouldn't _also_ use `NxList.Item` with them.

Reply:
> I'd be in favor of calling these `NxList.LinkItem` and `NxList.ButtonItem` or `NxList.ClickableItem` to make it clear that they encapsulate the `<li>` and that you shouldn't _also_ use `NxList.Item` with them.

These have been changed to `NxList.LinkItem` and `NxList.ButtonItem`.
12bca56fd2c81312c66eef2f7dfa14ba86b42e95